### PR TITLE
Move client-api to client folder and use ES6

### PR DESCRIPTION
**Pull request by mxstbr**
See #20

### CODE REVIEW

**Comment by mxstbr** [comment]
Would be nice to merge this before I go, could you review it @jantimon? Thx!

File: plugin/styleguide-plugin.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ StyleguidePlugin.prototype.apply = function apply(compiler) {
 
   compiler.plugin('emit', (compilation, callback) => {
     // Get the bundled Styleguide Client
-    const clientApi = fs.readFileSync(path.join(__dirname, 'client-api.js'));
+    const clientApi = fs.readFileSync(path.join(__dirname, '../client', 'client-api.js'));
```
Comment (jantimon) [inline plugin/styleguide-plugin.js]: This won't work as client-api is es6 now.
But its a good start point - I'll try to finish this tonight

Comment (mxstbr) [inline plugin/styleguide-plugin.js]: Oh yeah I totally forgot that won't work, damn

**Comment by nikgraf** [comment]
@mxstbr @jantimon can we close this down?

**Comment by mxstbr** [comment]
Nooo we should get this in, can you make this work Jan?

**Pull request closed without merging**